src/main/java/controller/LottoGameController.java

```diff
@@ -0,0 +1,89 @@
+package controller;
+
+import domain.lotto.Lotto;
+import domain.lotto.LottoBall;
+import domain.lotto.LottoBundle;
+import domain.money.GameMoney;
+import domain.result.LottoRank;
+import domain.result.LottoResult;
+import domain.result.WinningResult;
+import view.InputView;
+import view.OutputView;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class LottoGameController {
+    public void run() {
+        final GameMoney gameMoney = makeGameMoney();
+
+        final LottoBundle lottoBundle = makeLottoBundle(gameMoney);
+
+        final WinningResult winningResult = makeWinningResult();
+
+        makeLottoResult(lottoBundle, winningResult, gameMoney);
+    }
+
+    private GameMoney makeGameMoney() {
+        try {
+            OutputView.printGameMoneyRequest();
+            int userGameMoney = InputView.getGameMoney();
+            return new GameMoney(userGameMoney);
+        } catch (IllegalArgumentException e) {
+            OutputView.printErrorMessage(e.getMessage());
+            return makeGameMoney();
+        }
+    }
+
+    private LottoBundle makeLottoBundle(final GameMoney gameMoney) {
+        final LottoBundle lottoBundle = gameMoney.buyLotto();
+        OutputView.printLottoBought(lottoBundle);
+        return lottoBundle;
+    }
+
+    private WinningResult makeWinningResult() {
+        try {
+            final Lotto winningLotto = makeWinningLotto();
+            final LottoBall bonusBall = makeBonusBall();
+            return new WinningResult(winningLotto, bonusBall);
+        } catch (IllegalArgumentException e) {
+            OutputView.printErrorMessage(e.getMessage());
+            return makeWinningResult();
+        }
+    }
+
+    private Lotto makeWinningLotto() {
+        try {
+            OutputView.printWinningLottoRequest();
+            final List<Integer> winningLottoNumber = InputView.getWinningLotto();
+            return new Lotto(winningLottoNumber.stream()
+                    .map(number -> LottoBall.valueOf(number))
+                    .collect(Collectors.toList()));
```

Comment:
```suggestion
            final List<Integer> winningLottoNumber = InputView.getWinningLotto();
            return new Lotto(winningLottoNumber);
```

변환의 역할을 생성자로 넘겨준다면 어떠한 차이가 있을지 고민해보아요

Reply:
오... 컨트롤러에서 비즈니스 로직을 호출할 필요가 없어지겠네요! 
리팩토링 하면서 적용해보겠습니다 :)